tests/propertyBinding/impl/qml/PropertyBindingInterfaceQmlImplementation.qml

```diff
@@ -0,0 +1,68 @@
+/**********************************************************************
+**
+** Copyright (C) 2018 Luxoft Sweden AB
+**
+** This file is part of the FaceLift project
+**
+** Permission is hereby granted, free of charge, to any person
+** obtaining a copy of this software and associated documentation files
+** (the "Software"), to deal in the Software without restriction,
+** including without limitation the rights to use, copy, modify, merge,
+** publish, distribute, sublicense, and/or sell copies of the Software,
+** and to permit persons to whom the Software is furnished to do so,
+** subject to the following conditions:
+**
+** The above copyright notice and this permission notice shall be
+** included in all copies or substantial portions of the Software.
+**
+** THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+** NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+** BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+** ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+** CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+** SOFTWARE.
+**
+** SPDX-License-Identifier: MIT
+**
+**********************************************************************/
+
+import QtQuick 2.0
+import tests.propertybinding 1.0
+
+PropertyBindingInterfaceQMLImplementation {
+    id: root
+    Component.onCompleted: console.log("QML implementation is used.");
+
+    comboStr1 :cmb
+
+    ComboStructure {
+        id: cmb
+        iData: 100
+        sData: "binding"
+        bData : true
+    }
+
```

Comment:
function test () {
 cmb.iData = 1234;
TEST(comboStr1.iData == 1234);
}

Reply:
cmb.iData isn't accessible from testFunction , as its a private . So Instead of above code snippet I am changing it to 
as below in QMLBackend `Implementation


` 34 PropertyBindingInterfaceTestQMLImplementation {
 35     id: root
 36     Component.onCompleted: console.log("QML implementation is used.");
 37 
 38     comboStr2: comboStr1
 39 }
`

`  function test_intPropertyBinding() {
 39         api.comboStr1.iData = 99;
 40         compare(api.comboStr2.iData, 99);
 41     }
`